src/gwc/src/main/java/org/geoserver/gwc/layer/GeoServerTileLayerInfoImpl.java

```diff
@@ -266,28 +268,50 @@ public void setMetaTilingX(int metaTilingX) {
         checkArgument(metaTilingX > 0);
         metaWidthHeight[0] = metaTilingX;
     }
-    
+
+    /**
+     * @see GeoServerTileLayerInfo#getExpireCache()
+     */
+    @Override
     public int getExpireCache() {
         return expireCache;
     }
-    
+
+    /**
+     * @see GeoServerTileLayerInfo#setExpireCache(int)
+     */
+    @Override
     public void setExpireCache(int expireCache) {
         this.expireCache = expireCache;
     }
-    
+
+    /**
+     * @see GeoServerTileLayerInfo#getExpireCacheList()
+     */
+    @Override
     public List<ExpirationRule> getExpireCacheList() {
         return expireCacheList;
     }
-    
+
+    /**
+     * @see GeoServerTileLayerInfo#setExpireCacheList(List)
+     */
+    @Override
     public void setExpireCacheList(List<ExpirationRule> expireCacheList) {
         this.expireCacheList = expireCacheList;
     }
-    
+
+    /**
+     * @see GeoServerTileLayerInfo#getExpireClients()
+     */
     @Override
     public int getExpireClients() {
     	return expireClients;
     }
-    
+
+    /**
+     * @see GeoServerTileLayerInfo#setExpireClients(int)
+     */
```

Comment:
I usually consider the `@Override` enough on its own.  Until recently "@see X" javadocs like this would get in the way in Eclipse meaning you had to click through to get to the real docs instead of just inheriting the,  That's fixed now but this is still a bit redundant.

Reply:
Fair enough, that class already had some methods with `@see` and `@Override`, so I was mostly trying to make things consistent. In general I would agree that just `@Override` is sufficient.